src/main/java/org/jabref/gui/frame/MainMenu.java

```diff
@@ -303,6 +314,12 @@ private void createMenu() {
 
                 factory.createMenuItem(StandardActions.REDOWNLOAD_MISSING_FILES, new RedownloadMissingFilesAction(stateManager, dialogService, preferencesService.getFilePreferences(), taskExecutor))
         );
+
+        EasyBind.subscribe(preferencesService.getGrobidPreferences().grobidEnabledProperty(), enabled -> {
+            newLibraryFromPdfMenuItemOnline.setVisible(enabled);
+            newLibraryFromPdfMenuItemOffline.setVisible(!enabled);
```

Comment:
offline should be also available even if I have grobid enabled

Reply:
Done 😅